client/state/data-layer/wpcom/jetpack-onboarding/test/index.js

```diff
@@ -92,6 +96,25 @@ describe( 'saveJetpackOnboardingSettings()', () => {
 	} );
 } );
 
+describe( 'storeJetpackOnboardingSettings()', () => {
+	const dispatch = jest.fn();
+	const siteId = 12345678;
+	const settings = {
+		siteTitle: 'My Awesome Site',
+		siteDescription: 'Not just another WordPress Site',
+	};
+
+	test( 'should dispatch action that updates Redux state upon successful save request', () => {} );
+	storeJetpackOnboardingSettings( { dispatch }, { siteId, settings } );
+
+	expect( dispatch ).toHaveBeenCalledWith(
+		expect.objectContaining( {
```

Comment:
Maybe we should also verify the action type? Or use `updateJetpackOnboardingSettings( siteId, settings )` here because we're not expected to test the internals of `updateJetpackOnboardingSettings` (we do that in its own tests anyway)?

Reply:
> Maybe we should also verify the action type?

https://www.youtube.com/watch?v=uQQvEiRHHBc

> Or use `updateJetpackOnboardingSettings( siteId, settings )` here because we're not expected to test the internals of `updateJetpackOnboardingSettings` (we do that in its own tests anyway)?

https://www.youtube.com/watch?v=UoSVlSy8Ci0